parm/ocnicepost/ocean.csv

```diff
@@ -3,16 +3,11 @@
       'SST', 2 ,'Ct' , 'bilinear' , '', ''
       'SSS', 2 ,'Ct' , 'bilinear' , '', ''
     'speed', 2 ,'Ct' , 'bilinear' , '', ''
-     'ePBL', 2 ,'Ct' , 'bilinear' , '', ''
```

Comment:
Same question as above in the ice file for this and the other ocean variables removed.
Does the inclusion of these variables in some form interfere with the netcdf to grib2 processing?

Reply:
Please see answer above for ice.csv.